docs/components/radio.md

```diff
@@ -1,161 +1,163 @@
 # Radio
 
-**KRadio** - KRadio is a wrapper around a Kong styled radio input.
+KRadio is wrapper for native input type `radio` elements.
 
 <KCard>
+  <template #title>Selected: {{ radioValue }}</template>
```

Comment:
suggestion: can we remove this wrapper KCard? (expect this comment on every docs page; we want to display the components as-is without KCard impacting font sizes, etc.)